.circleci/config.yml

```diff
@@ -0,0 +1,124 @@
+version: 2.1
+
+orbs:
+  ruby: circleci/ruby@1.1.2
+  node: circleci/node@2
+  browser-tools: circleci/browser-tools@1.1
+  cloudfoundry: circleci/cloudfoundry@1.0
+  slack: circleci/slack@4.3.0
+
+jobs:
+  build:
+    docker:
+      - image: cimg/ruby:2.7-node
+      - image: redis
+    resource_class: medium
+    steps:
+      - checkout
+      - ruby/install-deps
```

Comment:
Presume this automatically caches?

Reply:
It does

https://circleci.com/developer/orbs/orb/circleci/ruby#commands-install-deps

Reply:
Nice